src/model/sql_query/row.rs

```diff
@@ -0,0 +1,190 @@
+// Copyright 2022 CeresDB Project Authors. Licensed under Apache-2.0.
+
+// [Row] in sql query
+
+use std::collections::BTreeMap;
+
+use arrow::{
+    array::{
+        ArrayRef, BinaryArray, BooleanArray, Int16Array, Int32Array, Int64Array, Int8Array,
+        StringArray, Time32MillisecondArray, TimestampMillisecondArray, UInt16Array, UInt32Array,
+        UInt64Array, UInt8Array,
+    },
+    datatypes::{DataType, TimeUnit},
+    record_batch::RecordBatch,
+};
+use paste::paste;
+
+use crate::{model::value::Value, Error, Result};
+
+#[derive(Debug)]
+pub struct Row {
+    // It is better to iterate in a fixed order, also can save memory.
+    values: BTreeMap<String, Value>,
+}
+
+impl Row {
+    pub fn column(&self, name: &str) -> Option<&Value> {
```

Comment:
rename `column` to `column_value`?
method `column` looks like get column property